app/views/HasNiphlsChangeView.scala.html

```diff
@@ -0,0 +1,63 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton
+)
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("hasNiphlsChange.title"))) {
+
+        <h1 class="govuk-heading-l">@messages("hasNiphlsChange.h1")</h1>
+
+            <p class="govuk-body">@messages("hasNiphlsChange.p1")</p>
+            <p class="govuk-body">@messages("hasNiphlsChange.p2")</p>
+
+        <div class="govuk-warning-text">
```

Comment:
can use the warning components as in the other PR https://github.com/hmrc/trader-goods-profiles-frontend/pull/214/files